### PR TITLE
Remove `example_output` for `ovc` and `ovc.convert_model`

### PR DESCRIPTION
**Pull request by slyalin**
Remove `example_output` added in https://github.com/openvinotoolkit/openvino/pull/17005. `output` parameter is dedicated to the declared purpose, there was no need for an extra parameter. The changes are applied for `ovc` part only, left unchanged in `mo` for backward compatibility.

TODO: @xczhai, please add more tests for this parameter used in this role (there is only one test with a single output).

### CODE REVIEW

File: tools/ovc/openvino/tools/ovc/convert_impl.py [hunk 1]
```diff
@@ -439,12 +439,11 @@ def _convert(cli_parser: argparse.ArgumentParser, args, python_api_used):
                 if 'example_input' in args and args['example_input'] is not None:
                     example_inputs = args['example_input']
 
-                #TODO: Check what example_outputs is and remove if not needed
                 example_outputs = None
-                if 'example_output' in args and args['example_output'] is not None:
-                    example_outputs = args['example_output']
+                if 'output' in args and args['output'] is not None:
+                    outputs = args['output']
```
Comment (xczhai) [inline tools/ovc/openvino/tools/ovc/convert_impl.py:444]: `outputs` here is duplicate of `example_outputs` in Line 442.

File: tools/ovc/openvino/tools/ovc/convert_impl.py [hunk 2]
```diff
@@ -439,12 +439,11 @@ def _convert(cli_parser: argparse.ArgumentParser, args, python_api_used):
                 if 'example_input' in args and args['example_input'] is not None:
                     example_inputs = args['example_input']
 
-                #TODO: Check what example_outputs is and remove if not needed
                 example_outputs = None
```
Comment (slyalin) [inline tools/ovc/openvino/tools/ovc/convert_impl.py:442]:
```suggestion
                outputs = None
```

**Comment by slyalin** [comment]
We have at least one test for that. Still need more tests: CVS-117808.

**Review by xczhai — APPROVED** [review]

**Review by popovaan — APPROVED** [review]

File: tests/layer_tests/mo_python_api_tests/test_mo_convert_paddle.py [hunk 3]
```diff
@@ -77,7 +77,7 @@ def create_paddle_static_module(tmp_dir):
     pd_model, x, y = make_pd_static_graph_model(shape)
     ref_model = make_ref_graph_model(shape)
 
-    return pd_model, ref_model, {"example_input": [x], "example_output": [y]}
+    return pd_model, ref_model, {"example_input": [x], "output": [y]}
```
Comment (popovaan) [inline tests/layer_tests/mo_python_api_tests/test_mo_convert_paddle.py:80]: OVC python API tests were moved to tests/layer_tests/**ovc**_python_api_tests/
So this change should be done here: https://github.com/openvinotoolkit/openvino/blob/600c2d82836ef9f9443b7db9e8d6502a71c4fcc3/tests/layer_tests/ovc_python_api_tests/test_paddle.py#L80

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Paddle.md [hunk 4]
```diff
@@ -40,7 +40,7 @@ Following PaddlePaddle model formats are supported:
 * ``paddle.fluid.dygraph.layers.Layer``
 * ``paddle.fluid.executor.Executor``
 
-Converting certain PaddlePaddle models may require setting ``example_input`` or ``example_output``. Below examples show how to execute such the conversion.
+Converting certain PaddlePaddle models may require setting ``example_input`` or ``output``. Below examples show how to execute such the conversion.
```
[the remark this answers is not in the document]
Comment (slyalin) [reply docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Paddle.md:44]: Agree.

**Comment by slyalin** [comment]
@xczhai, I've reverted the docs and tests, please add the necessary tests as agreed.

**Pull request merged**